tests/acceptance/features/coreApiWebdavLocksUnlock/unlockSharingToShares.feature

```diff
@@ -1,5 +1,8 @@
 @api @issue-1284
 Feature: UNLOCK locked items (sharing)
+  As a user
+  I want to lock the shared resource
+  So that share receiver cannot unlock the shared resource
```

Comment:
```suggestion
  As a user
  I want to unlocking a shared resource that has been locked by me to be restricted
  So that other users cannot unlock the shared resource
```